internal/clientcache/internal/cache/repository_refresh_token.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package cache
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/util"
+	"github.com/hashicorp/go-dbw"
+)
+
+// RefreshTokenValue is the the type for the actual refresh token value handled
+// by the client cache.
+type RefreshTokenValue string
+
+// lookupRefreshToken returns the refresh token stored in the db unless it is
+// to old in which case it does not return a refresh token.
```

Comment:
```suggestion
// too old in which case it does not return a refresh token.
```

Reply:
Does that mean that it returns an empty string? Or an error? Please expand this comment.